lib/coinbase/wallet.rb

```diff
@@ -17,6 +17,17 @@ class Wallet
     # The maximum number of addresses in a Wallet.
     MAX_ADDRESSES = 20
 
+    # A representation of ServerSigner status in a Wallet.
+    module ServerSignerStatus
+      # The Wallet is awaiting seed creation by the ServerSigner. At this point,
+      # the Wallet cannot create addresses or sign transactions.
+      PENDING = :pending_seed_creation
+
+      # The Wallet has an associated seed created by the ServerSigner. It is ready
+      # to create addresses and sign transactions.
+      ACTIVE = :active_seed
```

Comment:
Any reason these need to be symbols v.s. frozen strings?

That way we do not need to cast `.to_s` every time we compare?